src/document-creator/document-creator.mdx

```diff
@@ -0,0 +1,181 @@
+
+### `"defaults"` field
+
+The `defaults` field is an object that contains the settings that you have that are constant among the different documents of the same kind of file.
+
+Usually the document store address, token registry address, template, issuers, company logo, signatures, etc. are stored here, since they are a constant in the different documents of the same kind.
+
+Default value of the form should also be stored here.
+
+For example, you can store a default value for a `"title"` field here, this value will be used in the form shema as the value for the `"title"` field.
+
+An example of the `"defaults"` field:
+
+```json
+"defaults": {
+  "$template": {
+    "type": "EMBEDDED_RENDERER",
+    "name": "BILL_OF_LADING",
+    "url": "https://demo-cnm.openattestation.com"
+  },
+  "issuers": [
+    {
+      "identityProof": {
+        "type": "DNS-TXT",
+        "location": "isaactest.xyz"
+      },
+      "name": "Demo Issuer",
+      "tokenRegistry": "0x6D489a6A2cbcB76FcEeA26102d1273cd4d3B6b10"
+    }
+  ],
+  "name": "Demo bill of lading"
+}
+```
+
+#### `"$template"` field
+
+The `$template` field is an object that refers to the custom renderer, please refer to [creating document renderer](https://openattestation.com/docs/advanced/custom-renderer).
+
+#### `"issuers"` field
+
+The `issuers` is an array of the issuer object. The required fields for each issuer are `name`, `identityProof` and one of `documentStore` OR `tokenRegistry`.
+
+Transferrable document will use a [`tokenRegistry`](./token-registry), where as, a verifiable document will use a `documentStore`.
+
+The `name` field is a string that refers to the name of the issuer.
+
+The `tokenRegistry` field is a string that refers to the address in which you deploy the token registry.
+
+The `documentStore` field is a string that refers to the address in which you deploy the document store.
+
+The `identityProof` field is an object that refers to the issuer identity, please refer to [open attestation's identity Proof](https://openattestation.com/docs/advanced/identity-proofs) fro more information.
+
+### `"schema"` field
+
+The `"schema"` field is an object used to generate a form. It follows [JSON schema format.](https://json-schema.org/)
+
+The structure of the schema will be derived from the structure of the document you expect to build.
+
+It will probably be a subset of an already existing schema without the data that you expect to be constants accross your document.
+
+An example of the `"schema"` field:
+
+```json
+"schema": {
+  "type": "object",
+  "required": ["blNumber"],
+  "properties": {
+    "blNumber": {
+      "type": "string",
+      "title": "BL number"
+    },
+    "item": {
+      "type": "string",
+      "title": "Goods"
+    }
+  }
+}
+```
+
+### `"Attachments"` field
+
+This `"attachments"` field, which is a optional field, contains the information about the required attachments for this document.
```

Comment:
an optional ?

Reply:
our code handles it, but i think we should not say optional here, so i updated the sentence to indicate you can have the attachments or not.